services/app-web/src/pages/StateSubmission/Contacts/Contacts.tsx

```diff
@@ -189,273 +189,661 @@ export const Contacts = ({
 
     const errorSummaryHeadingRef = React.useRef<HTMLHeadingElement>(null)
     const [focusErrorSummaryHeading, setFocusErrorSummaryHeading] = React.useState(false)
-
-    /*
+const includeActuaryContacts =
+    draftSubmission.submissionType !== 'CONTRACT_ONLY'
+/*
      Set focus to contact name field when adding new contacts.
      Clears ref and focusNewContact component state immediately after. The reset allows additional contacts to be added and preserves expected focus behavior.
     */
-    React.useEffect(() => {
-        if (focusNewContact) {
-            newStateContactNameRef.current &&
-                newStateContactNameRef.current.focus()
-            setFocusNewContact(false)
-            newStateContactNameRef.current = null
-        }
-        if (focusNewActuaryContact) {
-            newActuaryContactNameRef.current &&
-                newActuaryContactNameRef.current.focus()
-            setFocusNewActuaryContact(false)
-            newActuaryContactNameRef.current = null
-        }
-    }, [focusNewContact, focusNewActuaryContact])
+React.useEffect(() => {
```

Comment:
It seems like prettier was never run on this file ? Or something ... sorry for the wild diff here